docs/Modules/bpmn-modules/modeling.md

```diff
@@ -9,4 +9,160 @@ parent: "BPMN Modules"
 grand_parent: "Modules"
 ---
 
-# modeling
\ No newline at end of file
+# modeling
+
+This module activates the BPMN 2.0 modeling features [over the common modeling module](/modules/common-modules/modeling). It means that this module contains every API exported by the common modeling module.
+
+Additional APIs over the common modeling module are:
+
+* [updateLabel](#updatelabel)
+* [connect](#connect)
+* [updateProperties](#updateproperties)
+* [resizeLane](#resizelane)
+* [addLane](#addlane)
+* [splitLane](#splitlane)
+* [setColor](#setcolor)
+
+## updateLabel
+
+`updateLabel` API updates labels and changes the names of elements. This API fires
+[elements.changed](/events/common-events/elements-changed) event and executes [element.updateLabel](/commands/bpmn-commands/element-updateLabel) command.
+
+### Usage
+
+```javascript
+const elementRegistry = bpmnJS.get('elementRegistry');
+const modeling = bpmnJS.get('modeling');
+
+const startEvent_1 = elementRegistry.get('StartEvent_1');
+
+modeling.updateLabel(startEvent_1, 'new label');
+```
+
+## connect
+
+`connect` API performs the same job as [the common createConnection API](/modules/common-modules/modeling#createconnection). The only difference is that `connect` API first checks if the source element and the target element are connectable by BPMN standards before delegating to `createConnection` API.
```

Comment:
Starting every explanation with the name of the method is redundant.

Reply:
Wording should be improved.

Maybe

```
Connects two shapes if allowed by the [BPMN rules](/modules/bpmn-modules/bpmn-rules). Uses [createConnection](/modules/common-modules/modeling#createconnection) internally.
```

Reply:
Done 👍